helm/nessie/values.yaml

```diff
@@ -37,6 +37,25 @@ authentication:
   # Set the OIDC client ID when authentication.enabled=true. Each application has a client ID that is used to identify the application
   oidcClientId: nessie
 
+# this is needed when selecting TRANSACTIONAL
+postgres:
+  secret:
+    name: postgres-creds
+    usernameKey: username
+    passwordKey: password
+  jdbcUrl: jdbc:postgresql://localhost:5432/my_database
+
+# JDBC configuration options for Quarkus https://quarkus.io/guides/datasource
+# quarkusJdbcOptions:
+#   - key: QUARKUS_DATASOURCE_JDBC_POOLING_ENABLED
+#     value: true
+
+# Sets the catalog name to use via JDBC
```

Comment:
given that we might want to expose more settingsI think it would make sense to move these settings into its own `versionStoreAdvancedConfig` section

Reply:
Sure.

Should I leave these ones or remove them and let someone configure them as part of `versionStoreAdvancedConfig` as needed?
```
# Sets the catalog name to use via JDBC
# versionStoreAdvancedTxJdbcCatalog: nessie

# Sets the schema name to use via JDBC
# versionStoreAdvancedTxJdbcSchema: nessie
```

Reply:
if they are not strictly necessary for now, then just leave them out